CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Updated terra-form-select dependency to ^6.0.0
+
+----------
```

Comment:
You can remove this dashed line, both this `###Changed` section and `###Fixed` section are gonna be part of the next release so there isn't a reason to separate them.

Reply:
https://github.com/cerner/terra-dev-site/pull/269/commits/18f412a45a219c8b2e9b3b61eddb801234989636#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR8 - Line fixed